tests/acceptance/bootstrap/NotificationContext.php

```diff
@@ -533,9 +534,10 @@ public function userShouldHaveReceivedTheFollowingEmailFromUserAboutTheShareOfPr
 	}
 
 	/**
-	 * @Then user :user should have received the following email from user :sender
+	 * @Then /^user "([^"]*)" should (not|)\s?have received the following email from user "([^"]*)"$/
```

Comment:
```suggestion
	 * @Then /^user "([^"]*)" (should|should not) have received the following email from user "([^"]*)"$/
```
I think this would be better as it is more readable.